microservices/api-gateway/vapt/values.yaml

```
@@ -0,0 +1,71 @@
+name: "interop-be-api-gateway"
+techStack: "nodejs"
+
+service:
+  create: true
+  albHealthCheck:
+    path: "/api-gateway/0.0/status"
+    protocol: "HTTP"
+
+serviceAccount:
+  roleArn: "arn:aws:iam::565393043798:role/interop-be-api-gateway-vapt-es1"
+
+configmap:
+  ACCEPTED_AUDIENCES: "{{.Values.namespace}}.interop.pagopa.it/m2m"
+  BASE_URL: "api.{{.Values.namespace}}.interop.pagopa.it"
+  API_GATEWAY_INTERFACE_VERSION: "0.0"
+  RATE_LIMITER_MAX_REQUESTS: "9999999"
+  RATE_LIMITER_BURST_PERCENTAGE: "1.0"
+  RATE_LIMITER_RATE_INTERVAL_MILLIS: "1000"
+  RATE_LIMITER_TIMEOUT_MILLIS: "300"
+  
+deployment:
+  livenessProbe:
+    httpGet:
+      path: "/api-gateway/0.0/status"
+      port: http
+    initialDelaySeconds: 15
+    periodSeconds: 5
+  readinessProbe:
+    httpGet:
+      path: "/api-gateway/0.0/status"
+      port: http
+    initialDelaySeconds: 15
+    periodSeconds: 5
+  env:
+    HOST: "0.0.0.0"
+    PORT: "8088"
+    SERVICE_VERSION: "2.0.25"
```

```suggestion
    SERVICE_VERSION: "2.0.25"
```
Could we use the image tag placeholder here? Would it work?